lib/uploadcare.rb

```diff
@@ -7,19 +7,19 @@
 
 module Uploadcare
   DEFAULT_SETTINGS = {
-      public_key: 'demopublickey',
-      private_key: 'demoprivatekey',
-      upload_url_base: 'https://upload.uploadcare.com',
-      api_url_base: 'https://api.uploadcare.com',
-      static_url_base: 'https://ucarecdn.com',
-      api_version: '0.5',
-      cache_files: true,
-      auth_scheme: :secure
-    }
+    public_key: 'demopublickey',
+    private_key: 'demoprivatekey',
+    upload_url_base: 'https://upload.uploadcare.com',
+    api_url_base: 'https://api.uploadcare.com',
+    static_url_base: 'https://ucarecdn.com',
+    api_version: '0.5',
+    cache_files: true,
+    store_files_upon_uploading: false,
```

Comment:
if we're releasing a major version, it's safe to break compatibility and default this to true (better behaviour) and make a not in changlog